src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -77,6 +82,9 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
                 return false;
             }
 
+            var cancellationToken = context.OperationContext.UserCancellationToken;
+            var docOptions = document.GetOptionsAsync(cancellationToken).WaitAndGetResult(cancellationToken);
```

Comment:
I would put docoption stuff inside of c => so that it can use right "c" (cancellation token)

Reply:
Done